qiskit/compiler/transpiler.py

```diff
@@ -561,128 +319,144 @@ def _parse_transpile_args(
     # If target is not specified and any hardware constraint object is
     # manually specified then do not use the target from the backend as
     # it is invalidated by a custom basis gate list or a custom coupling map
-    elif basis_gates is None and coupling_map is None:
-        target = _parse_target(backend, target)
+    elif basis_gates or coupling_map:
+        _skip_target = True
+    else:
+        target = getattr(backend, "target", None)
 
-    basis_gates = _parse_basis_gates(basis_gates, backend)
-    initial_layout = _parse_initial_layout(initial_layout, circuits)
-    inst_map = _parse_inst_map(inst_map, backend)
+    initial_layout = _parse_initial_layout(initial_layout)
     coupling_map = _parse_coupling_map(coupling_map, backend)
-    backend_properties = _parse_backend_properties(backend_properties, backend)
-    backend_num_qubits = _parse_backend_num_qubits(backend, num_circuits)
     approximation_degree = _parse_approximation_degree(approximation_degree)
+
     output_name = _parse_output_name(output_name, circuits)
```

Comment:
`output_name` still broadcasts in `_parse_output_name`, but that at least doesn't get passed to the `PassManager`, so I guess we're ok with keeping that?  I honestly had no idea we even had that option.

Reply:
Yeah it shouldn't matter for this, it felt like if we're taking an output name argument it needs to support broadcasting. Since if you did `transpile([qc1, qc2, qc3, qc4], output_name='foo')` I'm not sure what the expected behavior for that would be (or why we'd force it on people). Since the broadcasting doesn't effect the `PassManager` object or the parallel dispatch I figured it was fine to keep it.

Reply:
I'm fine with keeping it broadcasting, since it doesn't affect the `PassManager`. If the option wasn't already in `transpile`, I wouldn't have been in favour of adding it, haha - turns out that the initial use-case was to distinguish the same circuit transpiled with broadcast arguments (#2745).